go.mod

```diff
@@ -3,6 +3,8 @@ module github.com/vektah/gqlparser
 require (
 	github.com/agnivade/levenshtein v1.0.1
 	github.com/andreyvit/diff v0.0.0-20170406064948-c7f18ee00883
+	github.com/kisielk/godepgraph v0.0.0-20181003180210-9a9a3d47fba3 // indirect
```

Comment:
whats `godepgraph` in here for?

Reply:
wow, sorry. I removed it by `go mod tidy`